config/generation.php

```diff
@@ -1,319 +1,59 @@
 <?php
-$svgNormalization = static function (string $tempFilepath, array $iconSet) {
-    /** @var string $svgContent */
-    $svgContent = file_get_contents($tempFilepath);
-    // replace <svg width="48" height="48" viewBox="0 0 48 48" fill="none" with <svg fill="currentColor" stroke="none" viewBox="0 0 24 24" that are expected by blade-icons package
-    $svgContent = str_replace('<svg width="48" height="48" viewBox="0 0 48 48" fill="none"', '<svg fill="currentColor" stroke="none" viewBox="0 0 48 48" ', $svgContent);
-    file_put_contents($tempFilepath, $svgContent);
-};
 
-return [
```

Comment:
All this is removed since the output of the generator function is identical to this config array here.
Same results but with 300 some less lines of config array to maintain!

Reply:
Exactly